Symfony/CS/Tokenizer/TokensAnalyzer.php

```diff
@@ -211,7 +211,84 @@ public function isArrayMultiLine($index)
     }
 
     /**
-     * Check if there is a lambda function under given index.
+     * Returns the attributes of the method under the given index.
+     *
+     * @param int $index Token index of the method (T_FUNCTION)
+     *
+     * @return array {
+     *     @var int|null $visibility T_PRIVATE, T_PROTECTED or T_PUBLIC
```

Comment:
Sorry I overlooked that stuff. It must have not been in the spec when I read it as it was a year ago, or I just missed it, lol.